packages/node/src/main.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright 2020-2024 SubQuery Pte Ltd authors & contributors
 // SPDX-License-Identifier: GPL-3.0
 
-import { initLogger } from '@subql/node-core/logger';
```

Comment:
This and the yargs imports are done for a reason to not require other things that depend on the global logger